Documentation/gettingstarted/kata.rst

```diff
@@ -75,6 +75,16 @@ Deploy Cilium release via Helm:
              --namespace kube-system \\
              --set containerRuntime.integration=containerd
 
+.. warning::
+
+    Kata containers do not work with :ref:`host-services`, or with
+    :ref:`kube-proxy replacement <kubeproxy-free>` in strict mode. They should
+    be disabled with ``--set hostServices.enabled=false`` (default) and
+    ``--set kubeProxyReplacement=disabled`` (or ``partial``)
+
+    Both rely on socket-based load-balancing, which is not possible given that
+    Kata containers are virtual machines running with their own kernel.
+
```

Comment:
We could also add a reference here to https://github.com/cilium/cilium/issues/15437 to say something like `GitHub Issue 15437 tracks this limitation.` so users have a way to see whether the limitation is still live (+ interact on the issue), and we can also refer back later to see whether this warning is up to date.